hello.html

```diff
@@ -131,39 +194,6 @@
         var platform = 'unknown';
         var refObjectId = undefined;
 
-        window.onload = function () {
-            let inFSL = false;
```

Comment:
`inFSL` calculation was removed from the hello*.html apps.  I think maybe it should be reinstated, including the `console.log()` output below?